mypy/stubtest.py

```diff
@@ -1132,7 +1145,13 @@ def get_typeshed_stdlib_modules(custom_typeshed_dir: Optional[str]) -> List[str]
         if path.stem == "__init__":
             path = path.parent
         module = ".".join(path.relative_to(stdlib_dir).parts[:-1] + (path.stem,))
-        if module.split(".")[0] in packages:
+        if (
+            module.split(".")[0] in packages
```

Comment:
This check is now a bit suspect to me:

* It only checks the top-level name of the package.
* And it shouldn't be necessary, since the package name doesn't get added to `packages` if the module is not supported, due to the check above.